.github/workflows/python.yml

```diff
@@ -44,4 +44,5 @@ jobs:
       if: ${{ matrix.python-version == '3.7' }}
       run: |
         bash ./tests/integration/conversion.sh
+        bash ./tests/integration/bucket_to_explicit.sh
```

Comment:
Maybe

```
for test in tests/integration/*.sh; do
  bash "${test}"
done
```

Or perhaps have an `all.sh` script that calls all integration tests?